pkg/sql/types/types.go

```diff
@@ -2452,8 +2452,6 @@ func IsStringType(t *T) bool {
 // the issue number should be included in the error report to inform the user.
 func IsValidArrayElementType(t *T) (valid bool, issueNum int) {
```

Comment:
seems like we can remove this function now. unless you think it will be useful to track other types?

Reply:
I thought of this, but I figured if we do end up needing something like this again we'll have to remember both of these places and be sad. I think it's fine to leave this in.